beater/handlers.go

```diff
@@ -70,21 +73,17 @@ func healthCheckHandler(_ processor.Processor, _ Config, _ reporter) http.Handle
 		}))
 }
 
-func logHandler(h http.Handler) http.Handler {
+func errorHandler(code int, err error) http.Handler {
```

Comment:
this looks like this is the only way to send errors, but that is not true. how would you reuse this eg. for errors returned in `processRequestHandler`?

Reply:
`errorHandler` is just a convenience handler when there are handler chains that should always end up in an error (like the frontend handler when frontend is disabled).